docker-local/README.md

```diff
@@ -16,6 +16,36 @@ Integration tests
 - [`postman`](./postman)
 
 ## Context, Parties, Backend components
+The purpose of this setup is to run an example of mojaloop arrangement locally using docker-compose.
+
+Parties in the testing layout are:
+
+- _MojaloopHub_ is a set of microservices and their dependencies, including:
+    - [central-ledger](https://github.com/mojaloop/central-ledger): (mysql, objstore (mongoDB), kafka)
+    - [account-lookup-service](https://github.com/mojaloop/account-lookup-service) - als: (mysql-als)
+    - [central-settlement](https://github.com/mojaloop/central-settlement): (mysql-als, kafka)
+    - [quoting-service](https://github.com/mojaloop/quoting-service): (central-ledger, mysql-als, kafka)
+    - [transaction-request-service](https://github.com/mojaloop/transaction-requests-service): (cental-ledger)
+    - [ml-api-adapter](https://github.com/mojaloop/ml-api-adapter): wrapper to expose MojaloopHub
+  > configuration folder: [docker](./docker)
+
+- _PISP_  is a new party in Mojaloop system to represent Payment Initiate System Provider where its dependencies are:
+    - pisp-backend: [mojaloop-simulator](https://github.com/mojaloop/mojaloop-simulator)
+    - pisp-scheme-adapter: [scheme-adapter](https://github.com/mojaloop/sdk-scheme-adapter)
+    - pisp-redis: redisDB
+  > configuration folder: [pisp](./pisp)
+
+- _DFSP A_ is a Payer's bank account holder
```

Comment:
Yeah we have it elsewhere and I have found it kinda confusing...

Reply:
@kleyow, of course, the Payer/Payee designation is highly context dependant and it could bring some misunderstanding... I will simply drop off using these names if the context doesn't enforce it.

Reply:
Payer / Payee is the approved terminology, can we discuss what you think may cause misunderstanding? Here's the Glossary: https://github.com/mojaloop/mojaloop-specification/blob/master/documents/Glossary.md

In fact 'parties' is a term thats used to mean something else, may be thats a term that can be substituted (with something such as 'entity' or a similar word)

Reply:
I think Pawel explained it nicely. In that Payer/Payee are context dependent or. In the case of the PISP setup DFSP A and B aren't really designated to say use A to test sending and B to test receiving.

I have no major qualm about Payer/Payee terminology, just that at least I didn't initial envision A or B being denoted as Payer/Payee.

Again I'm fine with as it is. Just my daily ramblings lol.